helm/coredns-app/templates/configmap.yaml

```diff
@@ -15,7 +15,7 @@ data:
         {{- end }}
         errors
         health
-        kubernetes {{ .Values.cluster.kubernetes.domain }} {{ .Values.cluster.kubernetes.API.clusterIPRange }} {{ .Values.cluster.calico.CIDR }} {
+        kubernetes {{ .Values.cluster.kubernetes.clusterDomain }} {{ .Values.cluster.kubernetes.API.clusterIPRange }} {{ .Values.cluster.calico.CIDR }} {
```

Comment:
What's purpose of this change, especially knowing that domain is already nested under "cluster" (more accurately under cluster.kubernetes, but still)?

Reply:
This is to use the value defined in https://github.com/giantswarm/installations/blob/master/icarus/appcatalog/default-appcatalog-values.yaml

Reply:
we can change the value in the app or installation repo I guess. Any preference @corest ?

Reply:
I see rename happened in https://github.com/giantswarm/installations/pull/1081

Reply:
Yes, I mentioned in the message in the PR :)

Reply:
Change was done for consistency, yes. But if causes any issues with migrations - I don't care that much. If not - let's use `clusterDomain`